examples/api/texture-3d/app.js

```diff
@@ -170,7 +168,7 @@ export default class AppAnimationLoop extends AnimationLoop {
   }
 
   isSupported() {
-    return this.isDemoSupported;
+    return this.demoNotSupported;
```

Comment:
`!this.demoNotSupported`?

Is this used?

Reply:
Oh no, those methods aren't used any more.